src/routes/form/+page.svelte

```diff
@@ -23,4 +25,61 @@
 		<option value="9">Sem IX</option>
 		<option value="10">Sem X</option>
 	</select>
+	<label for="role">Roll No</label>
+	<input type = "text" id = "role" name = "role">
+	<label for="category">Category</label>
+	<select id="category">
+		<option value="1">General</option>
+		<option value="2">EWS</option>
+		<option value="3">OBC</option>
+		<option value="4">WDP</option>
+		<option value="5">SC</option>
+		<option value="6">ST</option>
+		<option value="7">VHC</option>
+		<option value="8">PHC</option>
```

Comment:
comprehensible `value`s would be better here: `gen`, `ews`, etc

Reply:
Didn't get you

Reply:
`<option value="gen">General</option>
<option value="ews">EWS</option>
...`